frontend/app/src/components/inputs/TagFilter.vue

```diff
@@ -93,5 +93,10 @@ export default class TagFilter extends Vue {
   &__clear {
     margin-top: -4px;
   }
+
+  /* stylelint-disable-next-line */
```

Comment:
Why disable? just do a `::v-deep { .v-select__slot {} }`

Reply:
I was getting an error when I initially tried this but now I see that it requires a double nest and that should fix it.

i.e.
```
::v-deep {
  .v-select {
    &__slot {
      ...
```

Reply:
Fixed